java/src/jmri/jmrit/display/layoutEditor/TrackSegment.java

```diff
@@ -1,15 +1,14 @@
 package jmri.jmrit.display.layoutEditor;
 
-import static jmri.jmrit.display.layoutEditor.LayoutTrack.NONE;
 import static jmri.util.MathUtil.add;
```

Comment:
"In other words, use it when you require _frequent_ access to static members from one or two classes."  (emphasis added)

I think the "frequent" part is the key.  If you're just referencing a member (method or constant) once or twice, the conceptual burden on the reader is reduced more if the class is explicit at point of reference:  `MathUtil.oneThirdPoint(..)`.   But if there are a lot of references to that, the reader has to wade through the extra text every time.  In that case, better to use the static import; the reader will figure out the reference once, remembering it from there on while reading lines that are more compact and easier to parse.  For this, the big about "import them individually" is key.  That means a quick search for the definition (even with a simple text editor) will find the import.